ice40/bitstream.cc

```diff
@@ -190,7 +200,7 @@ void configure_extra_cell(chipconfig_t &config, const Context *ctx, CellInfo *ce
         }
 
         value.resize(p.second);
-        if (p.second == 1) {
+        if ((p.second == 1) || !has_ec_cbit(bc, p.first + "_0")) {
```

Comment:
I think this also needs a check that bit 1 isn't set, or it would be a silent miscompilation of divide-by-5 using an older database.

Reply:
Well building with an older nextpnr already leads to a miscompilation right now.

Reply:
(although miscompilation for something that's unsupported and doesn't work under radiant or icecube might be a bit of a stretch)

Reply:
Ah yeah, in that case it doesn't really matter, I'm happy to merge once the CI is green.